components/layout/wrapper.rs

```diff
@@ -263,6 +265,10 @@ impl<'ln> ::selectors::Node<LayoutElement<'ln>> for LayoutNode<'ln> {
             _ => false
         }
     }
+
+    fn is_element_or_non_empty_text(&self) -> bool {
+        unimplemented!()
```

Comment:
This `unimplemented!()` is intentional because I'm not sure how this should be implemented for `LayoutNode`. If anyone has suggestions, please let me know